docs/dev/cut-a-new-release.md

```diff
@@ -1,38 +1,58 @@
-            ```
-            [Full Changelog](https://github.com/mozilla/glean/compare/v0.0.1...v0.1.0)
-            ```
-            Note that this needs three dots (`...`) between the two tags (two dots is different). Yes, the second tag doesn't exist yet, you'll make it later.
-        3. Optionally, go over the commits between the past release and this one and see if anything is worth including.
-        4. Make sure the changelog follows the format of the other changelog entries.
-            - Note that we try to provide PR or issue numbers (and links) for each change. Please add these if they are missing.
-
-    2. Add a new `# Unreleased Changes` on top
-
-2. Bump the versions
-    * Rust crates: Bump `version` in [`glean-core/Cargo.toml`](https://github.com/mozilla/glean/blob/master/glean-core/Cargo.toml) and [`glean-core/ffi/Cargo.toml`](https://github.com/mozilla/glean/blob/master/glean-core/ffi/Cargo.toml).  **NOTE:** There are two versions to update in `glean-core/ffi/Cargo.toml`.
-    * Kotlin package: Bump `libraryVersion` in the top-level [.buildconfig.yml](https://github.com/mozilla/glean/blob/master/.buildconfig.yml) file.
-    * Gradle plugin: Bump `project.ext.gleanVersion` in [`GleanGradlePlugin.groovy`](https://github.com/mozilla/glean/blob/master/gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy).
-    * Be sure you're following semver, and if in doubt, ask.
-    * Run `make build` to make sure the `Cargo.lock` file is updated.
-3. Land the commits that perform the steps above. This takes a PR, typically, because of branch protection on master.
+    1. Add any missing important changes under the `Unreleased changes` headline
+2. Run `bin/prepare-release.sh <new version>`
+    1. The new version should be the next patch, minor or major version of what is currently released.
+    2. Let it create a commit for you.
+3. Land the new commit that perform the steps above. This takes a PR, typically, because of branch protection on master.
 4. Cut the actual release.
     1. Click "Releases", and then "Draft a New Release" in the github UI.
-    2. Enter `v<myversion>` as the tag. In the example above it would be `v0.1.0`. It's important this is the same as the tags you put in the links in the changelog.
+    2. Enter `v<myversion>` as the tag. It's important this is the same as the tags you put in the links in the changelog.
     3. Under the description, paste the contents of the release notes from `CHANGELOG.md`.
     4. Note that the release is not available until the CI build completes for that tag.
         - You can check [on CircleCI for the running build](https://circleci.com/gh/mozilla/glean).
-5. After the release, bump the versions to the next pre-release:
-    * Rust crates: Use the next major version and append `-alpha.1`
-    * Kotlin package: Use the next major version and append `-SNAPSHOT`
```

Comment:
Just to confirm: we no longer need this step (to go back to alpha/SNAPSHOT)?  I agree it would be nice to skip it.  I guess it only matters if someone wanted to manually publish a release between tagged releases.  That should be *really* rare and you'd have to really know what you are doing anyway.

Reply:
@badboy does it make sense to keep this manual docs somewhere in the `internal` docs section, in case automation goes crazy?

Reply:
a "manual release between tagged releases" would be the workflow described further below.
I don't want these to happen off master anymore.

Not sure why we would need this anywhere internal. Having a "-SNAPSHOT" version should not be needed anymore on master at all